core/src/main/scala/cats/Show.scala

```diff
@@ -120,6 +117,10 @@ object Show extends ScalaVersionSpecificShowInstances with ShowInstances {
 private[cats] trait ShowInstances extends cats.instances.NTupleShowInstances with ShowInstances0 {
   implicit def catsShowForFiniteDuration: Show[FiniteDuration] =
     cats.instances.finiteDuration.catsStdShowForFiniteDurationUnambiguous
+
+  implicit def catsShowForSortedSet[A: Show]: Show[SortedSet[A]] = cats.instances.sortedSet.catsStdShowForSortedSet[A]
+  implicit def catsShowForSortedMap[K: Show, V: Show]: Show[SortedMap[K, V]] =
+    cats.instances.sortedMap.catsStdShowForSortedMap[K, V]
```

Comment:
It looks like now `Show` will be ambiguous. What if we move `catsShowForSet` and `catsShowForMap` up instead? Does it work?

Reply:
Also adding a test would be great (well not really a test, just summon the instance in test code - I think if you look for it there are places that test summoning some instance compiles).

Reply:
I'm not seeing an error trying to summon, e.g. `Show[SortedMap[Int, String]]` with this patch. Should I be looking at another test case?

Reply:
Also, I did not write that anywhere, but my change was largely motivated by the observation that Duration and FiniteDuration are like that and their implicits are OK

Reply:
Oh, my bad - you're right. The rule goes like this:
```scala
class DefinesA { implicit a: A }
class DefinesB { implicit b: B }
```

* if `A <:< B` then `a` gets a point in the implicit score
* if `DefinesB` <:< `DefinesA` then `b` gets a point in the implicit score
* so both would be 1:1 so ambiguous

However, `Show[SortedSet[A]]` is **not** a subtype of `Show[Set[A]]` because `Show` is invariant.
It just looked similar to issues we've had in the past like this one: #4569

Reply:
But this makes me think that it will pick the wrong instance for `SortedSet` and `SortedMap` - I guess that warrants adding a unit test.

Reply:
Also `BitSet` has the same issue

Reply:
> But this makes me think that it will pick the wrong instance for `SortedSet` and `SortedMap` - I guess that warrants adding a unit test.

It does seem to be the case (for implicitly[ContravariantShow[...]], since as you noted Show[...] cannot have that problem, being invariant).

I'll look into it more when I can, hopefully before Monday.

Reply:
The relationship between `SortedSet` and `Set` is the same as `List` and `Seq`, e.g.
And the instance for `Seq` is even further down, so the opposite - we moved `Seq`, not `List`